src/core/shape_inference/include/fft_common_validation.hpp

```diff
@@ -104,23 +104,23 @@ void validate_axes(const ov::op::util::FFTBase* op,
         axes_set.insert(static_cast<size_t>(axis));
     }
 
-    NODE_VALIDATION_CHECK(op, axes.size() == axes_set.size(), "(I)RDFT op axes must be unique.");
+    NODE_VALIDATION_CHECK(op, axes.size() == axes_set.size(), "Each axis must be unique.");
 }
 
 template <class T>
 void validate_signal_size(const ov::op::util::FFTBase* op, const T& axes_shape, const T& signal_size_shape) {
     NODE_VALIDATION_CHECK(op,
```

Comment:
As exception message has been modified, consider to use `NODE_SHAPE_INFER_CHECK` and reduce message strings by e.g. remove `signal_size_shape`.
Comment applies to all modified `NODE_VALIDATION_CHECK`

Reply:
Updated to pass `input_shapes` to the validation helpers and use `NODE_SHAPE_INFER_CHECK`.